index.html

```diff
@@ -1238,6 +1238,369 @@ <h2>Evidence</h2>
+				"nonce": "d61c4599-0cc2-4479-9efc-c63add3a43b2",
+				"signatureValue": "pYw8XNi1..Cky6Ed = "
+			}
+		}
+	],
+	"proof": [{
+		"type": "RsaSignature2018",
+		"created": "2017-06-18T21:19:10Z",
+		"creator": "did:example:76e12ec21ebhyu1f712ebc6f1z2/keys/2",
+		"nonce": "c0ae1c8e-c7e7-469f-b252-86e6a0e7387e",
+		"signatureValue": "BavEll0/I1..W3JT24 = "
+	}]
+}
+        </pre>
+
+
+   </section>
+  </section>
+
+
+
+   
+   <section>
+      <h2>Holder Acts on Behalf of Subject</h2>
+
+      <p>
+When the holder wants to act on behalf of the subject, the issuer MAY indicate this by
+either including the relationship of the holder in the subject's <a>claim</a>, 
+or by issuing a separate relationship credential to the holder. 
+Either way, the relationship describes the relationship between the holder and the subject of the claim. </p>
+
+<p>It is outside
+the scope of this standard how the issuer verifies the subject, the holder and the
+relationship between them.
+
+<p>
+Note: it is for further study which, if any, of the above mechanisms is to be 
+standardised by the working group.
+
+     </p>
+
+     
+        <pre class="example nohighlight" title="An example of the relationship 
+property in a child's credential, so that a parent can hold it">
+
+{
+	"id": "http://dmv.example.gov/credentials/3732",
+	"type": ["Credential", "ProofOfAgeCredential"],
+	"issuer": "https://dmv.example.gov/issuers/14",
+	"issued": "2010-01-01T19:73:24Z",
+	"claim": {
+		"id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+		"ageUnder": 16,
+		"parent": {
+			"type": "Mother",
+			"id": "did:example:ebfeb1c276e12ec211f712ebc6f"
+		}
+	},
+	"proof": {
+		"type": "RsaSignature2018",
+		"created": "2017-06-17T10:03:48Z",
+		"creator": "https://dmv.example.gov/issuers/14/keys/234",
+		"nonce": "d61c4599-0cc2-4479-9efc-c63add3a43b2",
+		"signatureValue": "pY9...Cky6Ed = "
+	}
+} 
+     </pre>
+
+<pre class="example nohighlight" title="An example of a relationship credential issued to
+a parent so that the parent can hold their child's verifiable credentials">
+
+{
+	"id": "http://dmv.example.gov/credentials/3732",
+	"type": ["Credential", "RelationshipCredential"],
+	"issuer": "https://dmv.example.gov/issuers/14",
+	"issued": "2010-01-01T19:73:24Z",
+	"claim": {
+		"id": "did:example:ebfeb1c276e12ec211f712ebc6f",
+		"child": {
+			"type": ["Person", "Child"],
+			"id": "did:example:ebfeb1f712ebc6f1c276e12ec21"
+		}
+	},
+	"proof": {
+		"type": "RsaSignature2018",
+		"created": "2017-06-17T10:03:48Z",
+		"creator": "https://dmv.example.gov/issuers/14/keys/234",
+		"nonce": "d61c4599-0cc2-4479-9efc-c63add3a43b2",
+		"signatureValue": "pY9...Cky6Ed = "
+	}
+}
+
+</pre>
+     
+<p> It can easily be seen from the above example how an authoritative issuer could indicate that a person is the owner of a pet dog </p>
+
+    </section>
+
+  <section>
+      <h2>Issuer Authorises Holder</h2>
+
+      <p>
+When the issuer wishes to authorise a holder to possess a credential that 
```

Comment:
I don't understand the use case here. Why does the law enforcement officer have this credential in the first place? It's not explained in the text.

Reply:
OOPS. I have omitted the figure that goes at the start of the section. I will need to add it.
(But if you look at the figure in the previous PR, which shows all the cases where Subject NE Holder, you will see that Law Enforcement is one of the examples where a holder may be given a VC by an issuer without the subject being aware of this, or not requesting it.)

Reply:
Look at this from the perspective of the verifier. The verifier is given a VC (for subject A) by holder B. The verifier needs to know (at least):
a) why has B got hold of this VC?
b) how is B related to A?
c) does B have A's permission to handle A's VC?
d) who to trust?
The answer to d) is the issuer. This is part of our trust model. Now if the issuer inserts into the VC that B is the holder of the VC and is related to A in the following way, then the verifier has all the above questions answered for him. This is why it is beneficial for some of the use cases for the issuer to insert information about the holder into the VC.

Reply:
Alternatively, if B is not related to A, then the verifier needs to know (at least)
a) why has B got hold of this VC?
b) how is B related to the issuer?
c) does B have the issuer's permission to handle A's VC?
d) who to trust.
Again, the verifier trusts the isssuer, and the issuer is saying that it gave the VC to the holder for it to present to verifiers, and that the holder is of the following type (which indicates the relationship between the issuer and the holder). So all the verifier's questions are answered.

@msporny. If the VC does not indicate who the holder is, and the verifier has to rely on the presentation (signed by the holder) to verify this, then I assert that this is not possible....
unless either:
a) the verifier modifies its trust policy (e.g. I trust X to hold this set of VCs) or
b) the holder presents some other credential proving that they have the right to hold the VCs of strangers.
I think what you are proposing unnecessarily complicates the work of either or both the verifier and the holder (and possibly the issuer as well).

Reply:
> d) who to trust?
> The answer to d) is the issuer. This is part of our trust model. 

Is it?  I don't understand this statement.  Anyone can say anything about anyone... VCs just give us a way to prove who's saying it.  Just because Bob issues a claim of foo about Carol in this VC doesn't mean that I trust that Bob's claim is true... I just know that Bob said it.  Deciding who I trust is a different step, internal to the program using the VC tooling, not part of the VC workflow itself, right?

Reply:
Please read Section 4, point 1
"The verifier trusts the issuer to issue the credential that it receives. "
If the verifier does not trust the issuer it will most likely discard the VC.